mcs_benchmark_data/pipelines/commonsense_qa/kagnet_commonsense_qa_submission_transformer.py

```diff
@@ -50,32 +54,35 @@ def __transform_submission(
 
             submission = json.loads(line)
 
-            if submission["name"].lower() != "kagnet":
-                continue
-
-            submission_obj = Submission(
-                uri="{}:submission:{}".format(self.__URI_BASE, submission["@id"]),
-                name=submission["@id"],
-                description=submission["description"],
-                date_created=submission["dateCreated"],
-                is_based_on=submission["isBasedOn"],
-                contributors=tuple(
-                    contributor["name"] for contributor in submission["contributor"]
-                ),
-                result_of=(
-                    submission["resultOf"]["@type"],
-                    strptime(submission["resultOf"]["startTime"], "%m-%d-%YT%H:%M:%SZ"),
-                    strptime(submission["resultOf"]["endTime"], "%m-%d-%YT%H:%M:%SZ"),
-                    submission["url"],
-                ),
-            )
+            if submission["name"].lower() == "kagnet":

```

Comment:
This should always be true, since you're in the Kagnet transformer. So you should be able to take out this check.

Reply:
It won't be, because that file contains submission data for all of the submissions (i.e. one line kagnet, one line roberta). Should I parse/separate it in the extractor?